pdfparser.js

```diff
@@ -117,7 +117,13 @@ export default class PDFParser extends EventEmitter {
 			nodeUtil.p2jinfo("PDF parsing completed.");
 			this.emit("pdfParser_dataReady", this.#data);
 		} else {
-			this.#data = { ...this.#data, ...data };
+			if(this.#data===data){
```

Comment:
when could this occur? this.#data is a collection of data from all data events.

Reply:
This was a mistake left in the commit, my bad. Removed